src/python/createLineFileDocsWithRandomLabel.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import os
+
+from random import randrange
+
+USAGE= """
+Usage: python createLineFileDocsWithRandomLabel.py
+
+"""
+
+ORIGINAL_LINEFILE = 'enwiki-20120502-lines-1k.txt'
```

Comment:
Would be nice to make these command-line args, e.g. `sys.argv[1]` and `[2]` :)  For now I am just locally editing it to run to generate new shared line-file-docs sources.

Reply:
Added ability to accept command line arguments in next commit.